addons/test_mail/tests/test_mail_activity.py

```diff
@@ -759,3 +761,253 @@ def test_week_grouping(self):
         self.assertEqual(groups[0][groupby], pg_groups["overdue"])
         self.assertEqual(groups[1][groupby], pg_groups["today"])
         self.assertEqual(groups[2][groupby], pg_groups["planned"])
+
+
+class TestActivitySchedule(TestActivityScheduleCommon):
```

Comment:
Would make a new file 'test_mail_activity_plan', holding plan- and scheduler- specific tests.